design-reference/architecture/authn/providers/azuread/integration_test.md

```diff
@@ -0,0 +1,50 @@
+# Purpose/Description
+
+For this design, an engineer building a template would like the ability to test a deployed application with authentication enabled. Currently the Cobalt test-harness cannot successfully complete http calls to an authenticated service and they will return a 401 unauthorized in the response. This is helpful in its self, because it does show that authentication has been configured. However, we would like the ability to also test for a 200 Successful status code. This will give the ability for the CI/CD pipeline to test for both negative and positive results during a deployment.
+
+There are two use cases we see an engineer would like to test:
+  1. Application Infrastructure deployment with  authentication enabled.
+  2. A deployed application with authentication enabled.
```

Comment:
Is this the app-dev flow?

Reply:
Great question and to the point I was trying to make... It can be used for both. I'll work on the wording a bit more.